docs/core_modules/model_modules/embeddings/usage_pattern.md

```diff
@@ -41,6 +41,8 @@ By default, embeddings requests are sent to OpenAI in batches of 10. For some us
 embed_model = OpenAIEmbedding(embed_batch_size=42)
 ```
 
+(local-embedding-models)=
```

Comment:
typo?

Reply:
This is for markdown ref links in sphinx

Reply:
Oh neat, good to know!